data/Jisan-mia.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "Jisan Mia",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Front-End Developer — JavaScript, React.js, Next.js",
+  "avatar": "https://github.com/Jisan-mia.png",
+  "tags": [
+    "NextJS",
+    "Typescript",
+    "Javascript",
+    "TailwindCSS",
+    "Open Source"
+  ],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/JisanMia4" },
+    { "platform": "github", "url": "https://github.com/Jisan-mia" }
+  ],
+  "links": [
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/JisanMia4",
+      "icon": "twitter"
+    },
+    {
+      "name": "GitHub: Collaborate on Open Source",
+      "url": "https://github.com/Jisan-mia",
+      "icon": "github"
+    },
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://linkedin.com/in/jisan-mia",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/jisan.mia4",
+      "icon": "instagram"
+    },
+    {
+      "name": "Medium: Share Learning",
+      "url": "https://jisanmia.medium.com",
+      "icon": "medium"
+    },
+    {
+      "name": "Dev.to",
+      "url": "https://dev.to/jisan",
+      "icon": "dev.to"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Frontend Developer",
+      "date": "December 2021",
+      "icon": "link",
+      "color": "grey",
+      "description": "Frontend Developer Full time Job",
```

You could change the wording here. Not necessary, but can make it look better, IMO. 🙂
```suggestion
      "description": "Full-time front-end developer job",
```